vimms/Controller/dia.py

```diff
@@ -81,8 +81,7 @@ def _process_scan(self, scan):
             # task.set(ScanParameters.CURRENT_TOP_N, 10) # time sampling fix see iss18
 
             scans.append(task)
-            #self.scan_number += 1 # increase every time we make a scan
-            #self.next_processed_scan_id = self.scan_number
+            self.scan_number += 1
             self.next_processed_scan_id += 2
```

Comment:
Still prefer this to be equal to `self.scan_number`

Reply:
Sure. will set next_processed_scan_id equal to scan_number